server/codegen.yml

```diff
@@ -6,11 +6,9 @@ generates:
       - typescript
       - typescript-resolvers
     config:
+      contextType: ../context#MyContext
       mappers:
-        # import { Message } from '../db'
```

Comment:
@MStokluska I noticed in a couple of PRs that comments are being removed. Is there any particular reason for that?

Reply:
@darahayes I like to comment code as guide as it gives me clearer picture of what a piece code purpose is, but I've realized that I need to practice every time I'm doing something new to get a better understanding of it. So I started a similar project on side where I practice all the new stuff and that's where I keep my comments.

Reply:
@MStokluska Cool, feel free to continue working on your side projects but I would ask you to keep those comments as you go along. They will help myself and @milenazuccarelli when we are reading through the code 🙂